app/validators/bookmark_controller_validator.rb

```diff
@@ -1,7 +1,10 @@
 class BookmarkControllerValidator < ActiveModel::EachValidator
-  @@active_record_tables = ActiveRecord::Base.connection.tables.map(&:to_s)
   def validate_each(record, attribute, value)
-    controllers = ["dashboard"] + (@@active_record_tables + Permission.resources.map {|x| x.tableize }).uniq
+    controllers = ["dashboard"] + (self.class.active_record_tables + Permission.resources.map {|x| x.tableize }).uniq
```

Comment:
would it make sense caching the entire controller list instead of generating it again on the fly each time?